package.json

```diff
@@ -58,9 +58,9 @@
     "mockdate": "^2.0.1",
     "prettier": "^1.11.1",
     "prop-types": "^15.6.1",
-    "react": "^16.2.0",
+    "react": "^16.3.0",
     "react-dom": "^16.2.0",
-    "react-styleguidist": "^6.5.1",
+    "react-styleguidist": "^7.0.3",
```

Comment:
Thank you for the updates. We should update `react-dom` and `react-test-renderer` as well, as they are released in tandem with one another.

Reply:
Will do.